get-readmes.sh

```diff
@@ -26,6 +26,7 @@ expressjs serve-static master
 expressjs session master
 expressjs timeout master
 expressjs vhost master
+expressjs express master/examles
```

Comment:
```suggestion
expressjs express master/examples
```

Reply:
thanks, force-pushed in order to not create multiple commits